homeassistant/components/alexa/handlers.py

```diff
@@ -911,7 +912,13 @@ async def async_api_adjust_target_temp(
             }
         )
     else:
-        target_temp = float(entity.attributes[ATTR_TEMPERATURE]) + temp_delta
+        current_target_temp: str | None = entity.attributes.get(ATTR_TEMPERATURE)
```

Comment:
During refactoring the `get` was removed, not expecting a `None` value. As it seems this can happen. This needs to be corrected.

Reply:
https://github.com/home-assistant/core/pull/97912